stable/superset/values.yaml

```diff
@@ -58,6 +58,9 @@ extraConfigFiles: {}
 ##
 extraEnv: {}
 
+## Extra arguments that will be passed to init_superset.sh
```

Comment:
There are two ways to solve this:
1) We can maintain to `values.yaml` files. One named `production-values.yaml` and one named `values.yaml`. 

2) The shortest path would be to create two examples in comments here, including one for `prod` and one for `dev`. That way new users of the helm chart won't need to go digging for right way for each scenario.